lib/src/lib/directives/form-field.directive.ts

```diff
@@ -9,25 +9,58 @@ import { FormFieldMappingService } from '../services/form-field-mapping.service'
 
 @Directive({
   selector: '[lab900FormField]',
-  standalone: true,
 })
 export class FormFieldDirective {
   private readonly container = inject(ViewContainerRef);
   private readonly formFieldMappingService = inject(FormFieldMappingService);
 
   public readonly schema = input.required<Lab900FormField>();
   public readonly group = input.required<UntypedFormGroup>();
+  public readonly fieldGroup = computed(() => {
+    const schema = this.schema();
+    if (schema?.attribute?.includes('.')) {
+      const attributeMap = schema?.attribute.split('.');
+      attributeMap.pop();
+      return this.group().get(attributeMap.join('.')) as UntypedFormGroup;
+    }
+    return this.group();
+  });
+
   public readonly language = input<string | undefined>(undefined);
   public readonly availableLanguages = input<ValueLabel[]>([]);
   public readonly readonly = input<boolean>(false);
+  public readonly fieldIsReadonly = computed(() => {
+    const options = this.schema().options;
+    if (this.readonly()) {
+      return true;
+    }
+    if (typeof options?.readonly === 'function') {
+      return options?.readonly(this.fieldGroup().value);
```

Comment:
is this truly reactive?
if the value changes, that doesn't mean the fieldGroup (object reference) changes

Reply:
the same applies for "options"
but maybe that's something you will fix in next pr anyway?

Reply:
options yes, value not so much but as it worked right now it didn't change if the value changed as well.